jenkins/run

```diff
@@ -50,6 +50,7 @@ podman run \
     --volume ${HOME_DIR} \
     ${GIT_BASE_VOLUME} \
     -w ${HOME_DIR}/pbench \
+    --env-host \
```

Comment:
Interesting. I'd considered using this, and wondered if we'd risk container pollution this way, particularly in a local container test run as opposed to the relatively "pristine" Jenkins environment. What specifically was behind this decision?

Reply:
Would it be better to transfer `COV_REPORT_LOC` specifically rather than throwing the entire pile over the fence? I'm not sure this is likely to be a real problem; just thinking in the abstract that keeping some isolation in the container is probably a good thing.